### DISCUSSION TITLE
Set legacy to false

### DISCUSSION THREAD

**Discussion by ochstobi** [Koenkk/zigbee2mqtt#6049] [Devices]
I have a question to the legacy flag.
Some devices recommend using legacy: false. But I can't find this flag elsewhere in the documentation. Where to put this? And what does it?

**Comment by Koenkk** [comment]
This disables the legacy integration for the device (if applicable). For some devices `click` was used which is now called `action`. Also the values of `action` could change because of this.

You can disable legacy for all devices by adding the following to your `configuration.yaml`.

```
device_options:
  legacy: false
```
